docs/source/usage/list.rst

```diff
@@ -0,0 +1,128 @@
+
+
+Output Format
+-------------
+
+The txt format for license information has the structure:
+::
+    Metadata component name:
+    Metadata component copyright 
+    Metadata component license 1 
+    Metadata component license 2 
+    ...
+
+    This product includes material developed by third parties: 
+
+    component 1 name:
+    component 1 copyright
+    component 1 license 1 
+    component 1 license 1 
+    ...
+
+    component 2 name:
+    component 2 copyright
+    component 2 license 1 
+    component 2 license 2 
+    ...
+
+
+The txt format for license information without the metadata component has the structure:
+::
+    component 1 name:
+    component 1 copyright
+    component 1 license 1 
+    component 1 license 1 
+    ...
+
+    component 2 name:
+    component 2 copyright
+    component 2 license 1 
+    component 2 license 2 
+    ...
+
+    
+The txt format for component information has the structure:
+::  
+    Metadata component name
+    Metadata component version
+    Metadata component supplier name
+
+    This product includes material developed by third parties: 
+
+    component 1 name
+    component 1 version
+    component 1 supplier name
+    
+    ...
+
+    
+The txt format for component information without the metadata component has the structure:
+::   
+    component 1 name
+    component 1 version
+    component 1 supplier name
+    
+    ...
+        
+
+The csv format for license information has the structure:
+::
+    Name,Copyright,Licenses
+    Metadata component name,Metadata component copyright,Metadata component license 1,...
+    component 1 name,component 1 copyright,component 1 license 1,component 1 license 2...
+    ...
+
+    
+The csv format for license information without the metadata component has the structure:
+::            
+    Name,Copyright,Licenses
+    component 1 name,component 1 copyright,component 1 license 1,component 1 license 2...
+    ...
+
+The csv format for component information has the structure:
+::     
+    Name,Version,Supplier
+    Metadata component name,Metadata component version,Metadata component supplier name
+    component 1 name,component 1 version,component 1 supplier name
+    ...
+
+
+The csv format for component information without the metadata component has the structure:
+::   
+    Name,Version,Supplier
+    component 1 name,component 1 version,component 1 supplier name
+    ...
+
+
+Examples::
+----------
+
+
+    # List the license information from bom.json including the metadata component
+    cdx-ev list licenses bom.json 
+
+    # List the license information from without the metadata component
+    cdx-ev list licenses bom.json --skip-metadata
+
+    # List the components from bom.json including the metadata component
+    cdx-ev list components bom.json 
+
+    # List the components from bom.json without the metadata component
+    cdx-ev list components bom.json --skip-metadata
```

Comment:
You need linebreaks between the comments and examples

Reply:
Removed the underline from Examples:: to create a code block